CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Added
+- `F*` class statistic
```

Comment:
I'm not sure if this is correct to say F* is added. What's your opinion? 
My suggestion is to mention in the Changed section as an update for the document of Jaccard index.

Reply:
Yeah, you are right!
@sadrasabouri